api/interservice/infra_proxy/request/nodes.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package chef.automate.domain.infra_proxy.request;
+option go_package = "github.com/chef/automate/api/interservice/infra_proxy/request";
+
+message AffectedNodes {
+    // ID of the Org.
+    string org_id = 1;
+    // ID of the Server.
+    string server_id = 2;
+    // The resource name (e.g. 'cookbooks', 'roles').
```

Comment:
should this be resource type? since name is right below?
are cookbooks and roles the only options?

Reply:
`resource_type` sounds good to me. oh wait it can be named as `chef_type` attribute as it gets returned detail API ref: https://docs.chef.io/api_chef_server/#get-24

Reply:
> are cookbooks and roles the only options?

It can be environments, policies, etc but for now, I have tested and verify for `cookbooks` and `roles`.

I think wroth putting?
```
// The resource name (e.g. 'cookbooks', 'roles', etc).
```

Reply:
> are cookbooks and roles the only options?

@susanev I Have also tested for `chef_environment`(environments & policy files) so `cookbooks`, `roles` and `chef_environment` only are the required attributes that need for infra views.

Updated the comments accordingly thanks!